tests/app/test_jinja_filters.py

```diff
@@ -3,39 +3,33 @@
 
 from mock import Mock
 
-from app.jinja_filters import format_date, format_conditional_date, format_currency, format_multilined_string, \
-     format_percentage, format_start_end_date, format_household_member_name, format_str_as_date, \
-     format_str_as_date_range, format_str_as_month_year_date, format_number_to_alphabetic_letter, format_unit
+from app.jinja_filters import format_date, format_conditional_date, format_currency, get_currency_symbol, \
+     format_multilined_string, format_percentage, format_start_end_date, format_household_member_name, \
+     format_str_as_date, format_str_as_date_range, format_str_as_month_year_date, format_number_to_alphabetic_letter, \
+     format_unit, get_formatted_number
 
 
 class TestJinjaFilters(TestCase):  # pylint: disable=too-many-public-methods
 
-    def test_format_currency_without_decimals(self):
-        # Given
-        currency = 1
-
-        # When
-        format_value = format_currency(currency)
-
-        self.assertEqual(format_value, '£1.00')
+    def test_get_formatted_number(self):
+        self.assertEqual(get_formatted_number('100.00', 2), '100.00')
+        self.assertEqual(get_formatted_number('1000'), '1,000')
+        self.assertEqual(get_formatted_number('10000'), '10,000')
+        self.assertEqual(get_formatted_number('100000000'), '100,000,000')
+        self.assertEqual(get_formatted_number('100.90', 2), '100.90')
```

Comment:
Needs tests added for:
- One decimal place in input, output defined as more - `self.assertEqual(get_formatted_number('100.1', 3), '100.100')`
- Output containing `,` and `.` - `self.assertEqual(get_formatted_number('10000', 2), '10,000.00')`